definitions/npm/nprogress_v0.2.0/test_nprogress.v0.2.0.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import nprogress from "nprogress";
+
```

Comment:
Bonus point for wrapping tests in `describe, it`. See https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope